e2e/suite_test.go

```diff
@@ -59,3 +64,30 @@ func TestMain(m *testing.M) {
 
 	os.Exit(testEnv.Run(m))
 }
+
+func dumpControllerLogs(n string) env.Func {
```

Comment:
@phoban01 @souleb  I added this as a debug. Do you guys think I should make this a thing in ocm-e2e-controller instead? Is it worth it, or should I just delete it?

Reply:
this would be valuable in the test framework as a wait to check status. But maybe not pods but and deployments? and Using a kubeClient instead of shelling out?

Reply:
Yeah, that's what I was thinking as well. I'll implement it there. :)

Reply:
Also, I don't mind using kubectl. It's a hell of a lot easier. This is a test framework. I see no point in complicating it. After all, the main Kuberentes e2e-framework is using kind's CLI to stop start kind clusters.

Reply:
But I'm open for change. Do you know any benefits about using the API instead of shelling out?

Reply:
the only advantage I see is that you don't need to install those binaries in your pipeline

Reply:
But you already have a Kubernetes cluster so 95% chance you already have kubectl. :D For us, it's a 100% since it's already required anyways. :D 

Anyways, I already wrote the code to stream logs so I can actually just copy it over I guess.

I'll use the library. :)